src/libraries/System.Speech/src/AudioFormat/EncodingFormat.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Speech.AudioFormat
+{
+    // These enumeration values are the same values used in the WAVEFORMATEX structure used in wave files.
+    public        enum EncodingFormat
```

Comment:
This formatting is weird. 
```suggestion
    public enum EncodingFormat
```